### DISCUSSION TITLE
How do I create phone a shortcut in Windows?

### DISCUSSION THREAD

**Discussion by airdmo** [srevinsaju/guiscrcpy#289] [Q&A]
I am using guiscrcpy-PyQt5-x64.exe.
How do I create a shortcut to my phone? (Using wireless connection).
I can't find a button for creating shortcuts in the gui.
I would be grateful for any help.
Thankyou.

**Comment by airdmo** [comment]
![圖片](https://user-images.githubusercontent.com/58673476/148170195-b109cfe7-4843-4b51-abb1-4d22678856dd.png)

**Reply by srevinsaju** [reply]
Actually, there is no shortcut system implemented for Windows yet.
